.github/workflows/community.yml

```
@@ -21,13 +20,13 @@ jobs:
       - uses: EddieJaoudeCommunity/gh-action-community/src/welcome@main
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          issue-message: '<h1>It''s great having you contribute to this project</h1> Feel free to raise an <strong>Issue</strong>! Welcome to the community :nerd_face:'
-          pr-message: '<h1>It''s great having you contribute to this project</h1> Feel free to create a <strong>Pull Request</strong>! Welcome to the community :nerd_face:'
-
+          issue-message: '<h1>It''s great having you contribute to this project</h1> Welcome to the community :nerd_face:'
+          pr-message: '<h1>It''s great having you contribute to this project</h1> Welcome to the community :nerd_face:'
+          footer: If you would like to continue contributing to open source and would like to do it with an awesome inclusive community, you should join our <a href="http://discord.eddiehub.org">Discord</a> chat and our <a href="http://github.eddiehub.org">GitHub Organisation</a>. We help and encourage each other to contribute to open source little and often 🤓 . Any questions? Let us know.'
```

```suggestion
          footer: If you would like to continue contributing to open source and would like to do it with an awesome inclusive community, you should join our <a href="http://discord.eddiehub.org">Discord Server</a> and our <a href="http://github.eddiehub.org">GitHub Organisation</a>. We help and encourage each other to contribute to open source little and often 🤓 . Any questions? Let us know.'
```
Can there be a small verbiage change(chat -> Server) and can `Discord Server` be hyperlinked for consistency since we hyperlink `Github Organisation`?